### DISCUSSION TITLE
Stepped gridlines

### DISCUSSION THREAD

**Discussion by markGilchrist** [patrykandpatrick/vico#803] [Questions]
### Question

I am writing an application that the will display an ECG (Echo Cardiogram) an example of which can be found here https://cvphysiology.com/arrhythmias/a009 

The grids line behind the graph need to be at set intervals so that the user can make assessments based on the data. 

my question is can we add major and minor lines to a graph?

### Vico version(s)

latest

### UI framework(s)

Jetpack Compose

**Comment by joseandroidengineer** [comment]
Do you mean 2 different line charts on the same graph?

**Comment by Gowsky** [comment]
Hello, @markGilchrist. `HorizontalAxis` and `VerticalAxis` don't offer separation into major and minor guidelines (grid). However, you can implement this using `Decoration` API. For example, you can use `HorizontalAxis` and `VerticalAxis` for major guidelines, and `Decoration` for minor guidelines.
